pkg/virt-operator/kubevirt.go

```diff
@@ -1151,3 +1152,28 @@ func (c *KubeVirtController) syncDeletion(kv *v1.KubeVirt) error {
 	logger.Info("Processed deletion for this round")
 	return nil
 }
+
+// checkIfConfigMapStillExists This function validates that the obsolete kubevirt-config configMap is no longer exist.
+// The user should not use this configMap to configure KubeVirt, and KubeVirt will ignore it anyway. In case the configMap
+// still exists, this function emit an event to notify the user, and ask them to delete it.
+func (c *KubeVirtController) checkIfConfigMapStillExists(logger *log.FilteredLogger, stopChan <-chan struct{}) {
+	ctx := context.Background()
+	getOpts := metav1.GetOptions{}
+
+	msg := fmt.Sprintf("the %s configMap is still deployed. KubeVirt does not support this configMap and it can be safely removed", obsoleteCmName)
+
+	go wait.Until(func() {
+		logger.V(5).Info("read the kubevirt-config configMap. if exist, emmit an event")
+		cm, err := c.clientset.CoreV1().ConfigMaps(c.operatorNamespace).Get(ctx, obsoleteCmName, getOpts)
+		if err != nil {
+			if errors.IsNotFound(err) {
+				logger.V(5).Info("The obsolete kubevirt-config configMap could not be found. good.")
```

Comment:
Should this stop?

Reply:
You mean the loop? I don't think so. Often people have no kubevirt config first and just the default settings and then they want to do their first modification and either post a configmap or change the CR. That is what we want to catch.

Reply:
Yes, sorry I apparently didn't wake up yet.